components/remote_settings/src/lib.rs

```diff
@@ -32,51 +32,6 @@ use storage::Storage;
 
 uniffi::setup_scaffolding!("remote_settings");
 
-/// The `RemoteSettingsContext` object represents the parameters and characteristics of the
-/// consuming application. For `remote-settings`, it is used to filter locally stored `records`.
-///
-/// We always fetch all `records` from the remote-settings storage. Some records could have a `filter_expression`
-/// attached to them, which will be matched against the `RemoteSettingsContext`.
-///
-/// When set, only records where the expression is true will be returned.
-#[derive(Deserialize, Serialize, Debug, Clone, Default, uniffi::Record)]
-pub struct RemoteSettingsContext {
-    /// Name of the application (e.g. "Fenix" or "Firefox iOS")
-    pub app_name: String,
-    /// Application identifier, especially for mobile (e.g. "org.mozilla.fenix")
-    pub app_id: String,
-    /// The delivery channel of the application (e.g "nightly")
-    pub channel: String,
-    /// User visible version string (e.g. "1.0.3")
-    #[serde(rename = "version")]
-    pub app_version: Option<String>,
-    /// Build identifier generated by the CI system (e.g. "1234/A")
-    pub app_build: Option<String>,
-    /// The architecture of the device, (e.g. "arm", "x86")
-    pub architecture: Option<String>,
-    /// The manufacturer of the device the application is running on
-    pub device_manufacturer: Option<String>,
-    /// The model of the device the application is running on
-    pub device_model: Option<String>,
-    /// The locale of the application during initialization (e.g. "es-ES")
-    pub locale: Option<String>,
-    /// The name of the operating system (e.g. "Android", "iOS", "Darwin", "Windows")
-    pub os: Option<String>,
-    /// The user-visible version of the operating system (e.g. "1.2.3")
-    pub os_version: Option<String>,
```

Comment:
I'm having doubt about removing `osVersion`. 
(Imagine a new iOS version offers a new feature or hardware capabilities that we would want to target in our products)
> Note: AFAIK we don't have it in Gecko, so it would have to be added there 👌